src/utils/xrQSlim/MxPropSlim.cpp

```diff
@@ -296,8 +296,7 @@ void MxPropSlim::collect_quadrics()
         case MX_WEIGHT_AREA:
         case MX_WEIGHT_AREA_AVG:
             Q *= Q.area();
-        // no break: fallthrough
-        default:
+        [[fallthrough]]; default:
```

Comment:
Please fix the code style for all files:
```suggestion
            [[fallthrough]];

        default:
```